src/tarantool_schema.c

```diff
@@ -261,19 +261,25 @@ parse_schema_space_value_value(struct schema_field_value *fld,
 	if (mp_typeof(**tuple) != MP_STR)
 		goto error;
 	const char *sfield = mp_decode_str(tuple, &sfield_len);
-	if (memcmp(sfield, "name", sfield_len) == 0) {
+	if (strncmp_strict(sfield, sfield_len, "name")) {
 		if (mp_typeof(**tuple) != MP_STR)
 			goto error;
 		sfield = mp_decode_str(tuple, &fld->field_name_len);
 		fld->field_name = pemalloc(fld->field_name_len, 1);
 		if (!fld->field_name)
 			goto error;
 		memcpy(fld->field_name, sfield, fld->field_name_len);
-	} else if (memcmp(sfield, "type", sfield_len) == 0) {
+	} else if (strncmp_strict(sfield, sfield_len, "type")) {
 		if (mp_typeof(**tuple) != MP_STR)
 			goto error;
 		sfield = mp_decode_str(tuple, &sfield_len);
 		fld->field_type = field_type_by_name(sfield, sfield_len);
+		if (fld->field_type == field_type_MAX)
+			goto error;
+	} else if (strncmp_strict(sfield, sfield_len, "is_nullable")) {
+		if (mp_typeof(**tuple) != MP_BOOL)
+			goto error;
+		fld->is_nullable = mp_decode_bool(tuple);
```

Comment:
AFAIU, it does not change any user visible behaviour? Just initializes the new fields of schema representation data structures. Correct? If so, I would try to make it more clear in the commit message.

BTW, please, structurize the commit message somehow: now it is single block of text and it is hard to deduce what is a list of changes, what is clarification of one of those changes, etc.

Reply:
> AFAIU, it does not change any user visible behaviour? Just initializes the new fields of schema representation data structures. Correct?

Yep

Reply:
> If so, I would try to make it more clear in the commit message.

Updated.

Reply:
> BTW, please, structurize the commit message somehow: now it is single block of text and it is hard to deduce what is a list of changes, what is clarification of one of those changes, etc.

Updated